### PR TITLE
Internationalize age; show age in months only up to 59 months.

### PR DESCRIPTION
**Pull request by zestyping**
We got a specific request from the MedCo here to show ages in months (18 months instead of 1 year 6 months).

### CODE REVIEW

File: app/src/main/res/layout/listview_tent_header.xml [hunk 1]
```diff
@@ -24,6 +24,6 @@
         android:layout_height="wrap_content"
         android:text="@string/unknown_tent"
         android:textAppearance="@style/text.heading"
-        tools:text="@string/probable" />
+        tools:text="Probable 1 (10 patients)" />
```
Comment (viniciusboson) [inline app/src/main/res/layout/listview_tent_header.xml:27]: Shouldn't it be internationalized ?

Comment (zestyping) [inline app/src/main/res/layout/listview_tent_header.xml:27]: The tools:text string is never shown to users -- it only appears as sample
text in the graphical UI layout editor in Android Studio.

Comment (G1DR4) [inline app/src/main/res/layout/listview_tent_header.xml:27]: ping, i`m fixing all the TODO i18n and also found even more strings to translate.

Comment (G1DR4) [inline app/src/main/res/layout/listview_tent_header.xml:27]: theres a feature/i18n branch

**Pull request merged**